### PR TITLE
Set sender email using `CTRL_ADMIN_EMAIL` env var

### PR DESCRIPTION
**Pull request by DoxasticFox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -445,3 +446,10 @@ git add .
 git commit -m [Your message](https://github.com/erlang/otp/wiki/writing-good-commit-messages)
 git push heroku master
 ```
+
+## <a id="environment-variables"></a> Environment Variables
+
+- `OTP_ENABLED` - Whether or not login should be permitted without entering a valid OTP. Valid values are `true` and `false`. Defaults to `true`.
+- `REDCAP_API_URL`
+- `REDCAP_CONNECTION_ENABLED` - Whether data should be forwarded to REDCap. Valid values are `true` and `false`. Defaults to `false`.
+- `REDCAP_TOKEN`
```
Comment (ignatiusm) [inline README.md:456]: Nice one adding more documentation :) You could possibly add a line here for the new `CTRL_ADMIN_EMAIL` env var too.

**Review by ignatiusm — CHANGES_REQUESTED** [review]
Looks good - I'm just setting up to test the changes locally, but assuming that goes well I only have the following suggestions:
- I made an comment on the README
- For devs that use Docker, it will be important that this new `CTRL_ADMIN_EMAIL` gets passed into the container. I suggest adding an entry to the `web` block of the `docker-compose.yml` file to pass the env var.

File: README.md (hunk 1 above)
Comment (DoxasticFox) [reply README.md:456]: Good point

**Comment by ignatiusm** [comment]
I'm just doing some work to test the changes locally, but one thing that I

**Comment by ignatiusm** [comment]
Easy to change `CTRL_ADMIN_EMAIL` env var and have it work in the email server. Looks good to me :+1: 

image

**Review by ignatiusm — APPROVED** [review]

**Comment by ignatiusm** [comment]
Thanks for adding the `makefile` tweak too! Nice one @DoxasticFox

**Pull request merged**